Docs (#28)
MaethorNaur/restui: README.md

### RestUI

> Once the React application build you can generate the package for RestUI

Reviewer: ```suggestion
Once the React application is built you can generate the package for RestUI
```

------------------------------------------------------------------------------------------------

### Kubernetes provider

In order to discover **OpenApi Specs** with RestUI in Kubernetes, the service **MUST** run inside the same Kubernetes

> cluster of your services.

Reviewer: One line?

The Kubernetes provider list and detect all running services in real time.

> New services are detected by polling from Kubernetes API at a regular interval.

Reviewer: ```suggestion
New services are detected by polling from the Kubernetes API at a regular interval.
```

------------------------------------------------------------------------------------------------

### Git provider

The git provider can be used to clone git repositories at a regular interval (`cache-duration`).
You can either provide the list of repositories you to clone or use Github to discover repositories.

It's possible to use both options at the same time.

Each option requires a list of `repositories`. This list can be either a **string** corresponding to

> the full url (`organization/project` for Github) or on an object.

Reviewer: ```suggestion
the full URL/Regex (`organization/project` for Github) or on an object.
```
